core/protocols/aws-xml-protocol/src/main/java/software/amazon/awssdk/protocols/xml/internal/marshall/XmlWriter.java

```diff
@@ -31,7 +31,16 @@
  * Utility for creating easily creating XML documents, one element at a time.
  */
 @SdkInternalApi
-final class XmlWriter {
+public class XmlWriter {
```

Comment:
Does this need to be public? S3XmlWriter is in the same package, right?

Reply:
removed public